src/apps/testapps/testH3IndexFat.c

```diff
@@ -1,148 +0,0 @@
-/*
- * Copyright 2017 Uber Technologies, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *         http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-/** @file
- * @brief tests H3 functions for manipulating H3 indexes
- *
- *  usage: `testH3IndexFat`
- */
-
-#include "h3Index.h"
-#include "h3IndexFat.h"
-#include "test.h"
-#include "utility.h"
-
-BEGIN_TESTS(h3IndexFat);
-
-TEST(h3FatIsValidWithMode) {
-    for (int i = 0; i <= 0xf; i++) {
-        H3IndexFat hf;
-        setH3IndexFat(&hf, 0, 0, 0);
-        hf.mode = i;
-        char failureMessage[BUFF_SIZE];
-        sprintf(failureMessage, "h3FatIsValid failed on mode %d", i);
-        t_assert(!h3FatIsValid(&hf) || i == 1, failureMessage);
-    }
-}
-
-TEST(h3FatNegativeResInvalid) {
```

Comment:
This one can't be tested on H3Index because there's only 4 bits for the resolution.